### DISCUSSION TITLE
immich home assistant addon - storage

### DISCUSSION THREAD

**Discussion by peeter22** [immich-app/immich#1949] [Q&A]
Hey guys.
I have recently found Immich and I am trying to set it up. 
I am trying to configure the Immich addon for home assistant running on a virtual machine on a windows PC.
I would like to assign a folder on a sepparate drive to be used as a photo storage folder.
I have tried tinker with different combinations of localdisks, networkdisks and data_location but I was not able to make it work.

Even if in the logs I see that D: drive was mounted successfully, after uploading a photo I can not find it anywhere on the D drive or inside the home assistant file structure.

Can someone please point me to a direction how to set this up ?
![Snímka obrazovky 2023-03-05 182652](https://user-images.githubusercontent.com/57701560/222976289-83adf96b-14bc-4b3d-a9e6-cf53018bbf9a.jpg)

**Comment by bo0tzz** [comment]
You'll probably need to ask whoever made that addon - we're not familiar with it at all, and can't provide support for it.
